packages/system/hetzner-robotlb/Makefile

```
@@ -0,0 +1,9 @@
+export NAME=hetzner-robotlb
+export NAMESPACE=kube-system
```

```suggestion
export NAMESPACE=cozy-hetzner-ccm
```
or its own namespace mayber?